File: Donation Rewards.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            if(x<=3)
            System.out.println("BRONZE");
            else if(x<=6)
            System.out.println("SILVER");
            else
            System.out.println("GOLD");
        }
    }
}

File: Donation Rewards.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int n;
        scanf("%d",&n);
        if(n<=3)
        printf("BRONZE\n");
        else if(n<=6)
        printf("SILVER\n");
        else
        printf("GOLD\n");
    }
}
